src/commands/deploy.ts

```diff
@@ -36,10 +37,11 @@ export default class Deploy extends Command {
   }
 
   validate_parameters(root_service: ServiceDependency, config_json: any) {
-    const res = { ...config_json };
+    const res = JSON.parse(JSON.stringify(config_json));
```

Comment:
why does this have to be done here?

Reply:
Clones the input so it doesn't get modified